pallets/did/src/errors.rs

```diff
@@ -84,7 +84,7 @@ pub enum SignatureError {
 /// Error generated when some extrinsic input does not respect the pallet's
 /// constraints.
 #[derive(Debug, Eq, PartialEq, TypeInfo)]
-pub enum InputError {
+pub enum Input {
```

Comment:
Name `Input` doesn't reflect that it's error codes.

Reply:
same as above